src/pytypes/onefuzztypes/models.py

```diff
@@ -568,6 +568,7 @@ class NodeCommandEnvelope(BaseModel):
 
 class Node(BaseModel):
     timestamp: Optional[datetime] = Field(alias="Timestamp")
```

Comment:
So logically, what does this mean? Is it `last_modified`, `last_heartbeat`, `last_event`, &c? It'd be good to document exactly what the meaning is, since we've been tripped up by changes that made it ambiguous.

Reply:
I'd say it's last_modified... and really, it's 'last time the storage table was modified'.

So maybe 'config_last_modified'

Reply:
Great, I think the comment will make it clear.

Reply:
Note that this field is not specific to this entity. This is a field is provided by Azure Table and is used by other entities too.
I don't think it is necessary to comment on its meaning here only since it is not logically different from any place where it is being used.

Reply:
Ah, that's right. Since we're just inheriting it from Azure Table, I agree with @chkeita on this field.